### PR TITLE
Even more ship names!

### PR DESCRIPTION
**Pull request by Pointedstick**
Lots of changes here:
- More words in all Civilian and Pirate categories
- Hugely simplified Civilian naming system, to make it less intimidating to others
- Extensively tested and tweaked to get rid of a lot of awkward combinations (like "Lucky Old")

### CODE REVIEW

File: data/names.txt [hunk 1]
```diff
@@ -807,40 +922,129 @@ phrase "standalone civilian ship names"
...
+	phrase
+		"male names"
 		"male names"
+		"male titles"
+		"female names"
 		"female names"
+		"female titles"
+		"gender-neutral titles"
+	word
+		"'s "
```
Comment (Pointedstick) [inline data/names.txt]: All the possessive titles and names were simplified to go here.

File: data/names.txt [hunk 2]
```diff
@@ -2158,6 +2159,7 @@ phrase "pirate"
 		"Bony Fingers"
 		"Yo Ho Ho"
 		"Low Blow"
+		"bilgewater"
```
Comment (KiLEdEnNis) [inline data/names.txt]: lacks a capitalized letter, should be "Bilgewater"

Comment (Pointedstick) [reply data/names.txt]: Already fixed in the latest commit!

Comment (KiLEdEnNis) [reply data/names.txt]: oops, my bad :P

**Pull request merged**